scripts/dev.ts

```diff
@@ -6,12 +6,18 @@
 
```

Comment:
Not sure where this comment should go, but it looks like something in this PR breaks `npm run dev` on Windows:
```
rancher-desktop> npm run dev

> rancher-desktop@1.6.0 dev
> node scripts/ts-wrapper.js scripts/dev.ts

C:\Users\adamk\dev\rancher-desktop\node_modules\.bin\nuxt:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1032:15)
    at Module._compile (node:internal/modules/cjs/loader:1067:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
C:\Users\adamk\dev\rancher-desktop\scripts\dev.ts:28
    const newError: Error & { code?: number } = new Error(combinedMessage);
                                                ^
Error: Renderer process error: Process exited with code 1
    at DevRunner.emitError (C:\Users\adamk\dev\rancher-desktop\scripts\dev.ts:28:49)
    at C:\Users\adamk\dev\rancher-desktop\scripts\dev.ts:53:28
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
Caused by: Error: Process exited with code 1
    at ChildProcess.<anonymous> (C:\Users\adamk\dev\rancher-desktop\scripts\lib\build-utils.ts:88:18)
    at ChildProcess.emit (node:events:526:28)
    at ChildProcess.emit (node:domain:475:12)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12) {
  code: undefined
}
```

Reply:
Good catch and thanks for testing in Windows to identify this one. Looks like this issue was related to the change where I updated 'node_modules/nuxt/bin/nuxt.js' to 'node_modules/.bin/nuxt'. I originally did this to more closely match the `@rancher/shell` implementation but it appears that this was not the right choice.

Reply:
Interestingly enough, we get a whole host of new errors in attempting to switch back. This will require some more investigation 🤔

```
ERROR  Failed to compile with 20 errors                                                                          
These dependencies were not found:                                                                                                                                                                                                  

* ..\node_modules\@rancher\shell\middleware\node_modules\@rancher\shell\middleware\authenticated.js in ./.nuxt/middleware.js
* ..\node_modules\@rancher\shell\middleware\node_modules\@rancher\shell\middleware\i18n.js in ./.nuxt/middleware.js
* ..\node_modules\@rancher\shell\middleware\node_modules\@rancher\shell\middleware\unauthenticated.js in ./.nuxt/middleware.js
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\action-menu.js in ./.nuxt/store.js       friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\auth.js in ./.nuxt/store.js              friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\aws.js in ./.nuxt/store.js               friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\catalog.js in ./.nuxt/store.js           friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\digitalocean.js in ./.nuxt/store.js      friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\features.js in ./.nuxt/store.js          friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\github.js in ./.nuxt/store.js            friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\growl.js in ./.nuxt/store.js             friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\i18n.js in ./.nuxt/store.js              friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\index.js in ./.nuxt/store.js             friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\linode.js in ./.nuxt/store.js            friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\plugins.js in ./.nuxt/store.js           friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\prefs.js in ./.nuxt/store.js             friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\resource-fetch.js in ./.nuxt/store.js    friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\type-map.js in ./.nuxt/store.js          friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\uiplugins.ts in ./.nuxt/store.js         friendly-errors 16:15:55
* ..\node_modules\@rancher\shell\store\node_modules\@rancher\shell\store\wm.js in ./.nuxt/store.js                friendly-errors 16:15:55
```

Reply:
@adamkpickering Windows issues should be resolved now with 1ba50ffe49d75a66805ae9ae641c57087c756957